config/default.yaml

```
@@ -107,6 +107,24 @@ commands:
   additionalPrefixes:
     - "mjolnir_bot"
 
+# Configuration specific to certain toggleable protections
+protections:
+  # Configuration for the wordlist plugin, which can ban users based if they say certain
+  # blocked words shortly after joining.
+  wordlist:
+    # A list of words which should be monitored by the bot.  These will match if any part
+    # of the word is present in the message in any case.  e.g. "poop" also matches
+    # "poOPYHEad".  Additionally, regular expressions can be used.
+    words:
+      - "nigger"
+      - "faggot"
+      - "tranny"
+      - "retard"
```

For better or worse, we try to keep this project neutral to any particular kinds of abuse. We also definitely do not try and expose admins of the bot to abuse themselves, even if it is an example.

```suggestion
    # A list of words which should be monitored by the bot. These will match if any part
    # of the word is present in the message in any case. Regular expressions are also supported.
    words:
      - "CaSE"
      - "inSEnsITIVe"
```